lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1844,6 +1845,18 @@ void InterpreterFunction::fwdDebugPrintInst(const DebugPrintInst *I) {
 //                Instructions used by Quantization
 //===----------------------------------------------------------------------===//
 
+template <class eTy = int8_t>
+static void quantizeTensor(Tensor *dest, Tensor *src) {
+  auto srcH = src->getHandle<float>();
+  auto destH = dest->getHandle<eTy>();
+  TensorQuantizationParams params{dest->getType().getScale(),
+                                  dest->getType().getOffset()};
+
+  for (size_t i = 0, e = destH.size(); i < e; ++i) {
+    destH.raw(i) = quantization::quantize<eTy>(srcH.raw(i), params);
+  }
+}
```

Comment:
Aren't we duplicating the logic of the quantization library here?
If not, could we have that there?